.codecov.yml

```diff
@@ -0,0 +1,9 @@
+---
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  status:
+    project:
+      default:
+        target: auto
```

Comment:
Hey @dblock can you add the file in this format https://github.com/opensearch-project/opensearch-build/blob/main/.codecov.yml ?

Reply:
This file is in the same format.